rpc/nodeagent/node/host/moc_nodeagent_host.proto

```diff
@@ -31,10 +31,12 @@ message Host {
 	string id = 2;
 	Status status = 3;
 	string certificate = 4 [(sensitive) = true];
-	int32  port = 5;
-	int32  authorizerPort = 6;
-	HostState runningState = 7;
-	Tags tags = 8;
+	string rootCACertificate = 5 [(sensitive) = true];
```

Comment:
Move rootCA and intermediateCA to repsonse

Reply:
Done